bindings/bind_py_representation_calculator.cc

```diff
@@ -0,0 +1,146 @@
+/**
+ * @file   bind_py_representation_calculator.cc
+ *
+ * @author Felix Musil <felix.musil@epfl.ch>
+ *
+ * @date   30 Oct 2018
+ *
+ * @brief  File for binding the Representation calculator
+ *
+ * Copyright  2018  Felix Musil, COSMO (EPFL), LAMMM (EPFL)
+ *
+ * Rascal is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3, or (at
+ * your option) any later version.
+ *
+ * Rascal is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GNU Emacs; see the file COPYING. If not, write to the
+ * Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+ * Boston, MA 02111-1307, USA.
+ */
+
+#include "bind_py_representation_calculator.hh"
+
+namespace rascal {
+
+  template <class Calculator>
+  using PyCalculator = py::class_<Calculator, CalculatorBase>;
+
+  template <typename Calculator>
+  decltype(auto) add_representation_calculator(py::module & mod, py::module &) {
```

Comment:
unused argument omitted again -- again, I don't understand why it is needed.

Reply:
at some point one might need to add the some stuff in the _internals. I found this convenient on several occasion